pubspec.yaml

```
@@ -33,3 +35,5 @@ dev_dependencies:
 
 flutter:
   uses-material-design: true
+  assets:
+    - assets/images/btn_google_signin/light/normal.png
```

[https://flutter.dev/docs/development/ui/assets-and-images](ここ)によると今後画像が増える可能性を考慮すると
```suggestion
    - assets/images/
```
って書けそう

---

この部分については、
`assets/images/`って最初していたんだけど、動作が不安定になることが多くて一旦fullPath指定にしてる。
fullPathを残したまま追記するね！

---

そんなことあるんね！興味深い！OK!!